api/tasks.py

```diff
@@ -11,3 +13,22 @@ def create_delivery(order):
     else:
         delivery.print_pdf()
         return {'success': 'all good'}
+
+
+@app.task
+def check_customer_is_partner(customer, created=True):
+    crm_client = CrmClient()
+    try:
+        is_partner = crm_client.check_customer_is_partner(customer)
+    except CrmError:
+        return {"error": "Error connecting to CRM", "customer": customer}
+
+    if is_partner:
+        # here will come the code to update the Wordpress user role.
+        pass
+
+    # in case we will do daily cronjobs from the wordpress users to update to false their role
+    # if their partner status has been removed from the CRM
+    if not created:
```

Comment:
are we going to implement this anytime soon? I bet not so I wouldn't this placeholder. We don't know how things are going to evolve so IMO there's no point in predicting it.